### ISSUE TITLE
Donut Chart - color change request

### ISSUE THREAD

**Issue by serenamarie125** [patternfly/patternfly-3#106]
When Donut Charts use thresholds, currently blue when the value has not reached warning or error.  Based on usability testing, the blue should be changed to green.  Please change it to HEX 3f9c35, per Kyle's specification.

Note that when no thresholds are used, the current blue color should still be used.

This needs to be changed in the PF site @rhamilto @LHinson for both Donut Chart & Utilization Chart.
@dtaylor113 once it's changed, this needs to be put into angular as well.

**Comment by LHinson** [comment]
Currently there are no visuals for thresholds on https://www.patternfly.org/patterns/donut-chart/. If we are going to include threshold color examples, we should also add visuals for the threshold indicators while we are at it. Documentation currently states visuals for thresholds are TBD. @serenamarie125

**Comment by kybaker** [comment]
I am not clear how thresholds should be visualized. Shouldn't we just change the blue to green here? I was under the impression showing x of x then having the donut change color was a threshold. https://www.patternfly.org/patterns/donut-chart/

**Comment by LHinson** [comment]
@kybaker see #5 and #6 here: https://www.patternfly.org/patterns/donut-chart/ The inidcators are described as "tick marks" on the chart in the docs. 

That way, the user has an understanding of when they are approaching the threshold instead of simply and abruptly changing the color on them.

**Comment by dtaylor113** [comment]
This is what we show in our angular-patternfly ng-docs:

[image: Inline image 1]

[image: Inline image 3][image: Inline image 4]
Where I think we want this last "Ok" to be green instead of blue.  Blue
should only be shown when there are no thresholds set.
- Dave

On Wed, Sep 2, 2015 at 1:45 PM, Kyle Baker notifications@github.com wrote:

> I am not clear how thresholds should be visualized. Shouldn't we just
> change the blue to green here? I was under the impression showing x of x
> then having the donut change color was a threshold.
> https://www.patternfly.org/patterns/donut-chart/
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/patternfly/patternfly/issues/106#issuecomment-137184895
> .

## 

Dave Taylor
Principle Software Engineer
(978) 392 - 1016

**Comment by dtaylor113** [comment]
The "tick marks" have not been implemented yet, but I think Serena is just
talking about changing the usage arc from blue to green.
- Dave

On Wed, Sep 2, 2015 at 2:11 PM, Leslie notifications@github.com wrote:

> @kybaker https://github.com/kybaker see #5
> https://github.com/patternfly/patternfly/issues/5 and #6
> https://github.com/patternfly/patternfly/pull/6 here:
> https://www.patternfly.org/patterns/donut-chart/ The inidcators are
> described as "tick marks" on the chart in the docs.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/patternfly/patternfly/issues/106#issuecomment-137195194
> .

## 

Dave Taylor
Principle Software Engineer
(978) 392 - 1016

**Comment by LHinson** [comment]
Preferably, I'd like to address all the updates (color and tick marks). Otherwise, I will be revisiting this page again in the near future. Plus, it would be nice to have visuals for the documentation that is there vs "(visuals tbd)"

Are we blocked on creating the visuals for tick marks for some reason @serenamarie125 @kybaker ?

**Comment by kybaker** [comment]
@LHinson if @serenamarie125 thinks we need them and she has not already designed something I will add them to the list to work on.

**Comment by serenamarie125** [comment]
@lhinson, I disagree about waiting to change the color until we have a decision on threshold indicators.  Although we had input saying that many wanted them, we need to come up with some designs and test them.  Much more time consuming than changing an RGB :)

I'll add a trello card for threshold indicators.

**Comment by serenamarie125** [comment]
@dtaylor113, yes I am talking about the full color of the usage arc in the chart.  Sorry if there was confusion.

**Comment by LHinson** [comment]
- Utilization Card [1] and Donut charts [2] visuals have been updated. 
- Donut Chart [2] docs have also been updated. 

[1] https://www.patternfly.org/patterns/utilization-card/
[2] https://www.patternfly.org/patterns/donut-chart/

Thanks!

**Comment by LHinson** [comment]
Also see: https://www.patternfly.org/whats-new/

**Comment by dtaylor113** [comment]
@dtaylor113 closed #106 via 5d1dbc4f546219d33eb0943f599ba8148d1ec59b
